python/xorbits/_mars/tests/core.py

```diff
@@ -326,7 +327,7 @@ def assert_tensor_consistent(self, expected, real):
             raise AssertionError(
                 f"Type of real value ({type(real)}) not one of {np_types!r}"
             )
-        if not hasattr(expected, "dtype"):
+        if not hasattr(expected, "dtype") or isinstance(real, NAType):
```

Comment:
Don't need to check if expected is NAType?

Reply:
Fix. Check expected's shape